src/routes/(settings)/settings/+page.svelte

```
@@ -1,16 +1,24 @@
 <script lang="ts">
   import { useCurrentUser } from "$lib/client/hooks/useCurrentUser.svelte";
-  import { ArrowRightIcon, Loader2Icon } from "@lucide/svelte";
+  import { ArrowRightIcon, Loader2Icon, CoinsIcon } from "@lucide/svelte";
```

ESLint

[error] 'CoinsIcon' is defined but never used.

(@typescript-eslint/no-unused-vars)